elyra/metadata/tests/test_handlers.py

```diff
@@ -273,12 +274,40 @@ def test_update_instance(self):
         instance = r.json()
         assert instance['metadata']['number_range_test'] == 7
 
+        # Now attempt the update again, but with bad metadata and ensure previous still exists
```

Comment:
not a big deal but i would advocate for breaking this up into multiple test methods each focusing one task (i.e., update non-existent, update valid instance, update bad metadata)

Reply:
You're making @lresende proud! :smile: Sounds good.